File: 3226-MinimumNumberGame/3226-MinimumNumberGame.py
# Last updated: 7/3/2026, 12:49:53 PM
class Solution:
    def numberGame(self, nums: List[int]) -> List[int]:
        a=[]

        while(nums):
            alice = nums.pop(nums.index(min(nums)))
            bob = nums.pop(nums.index(min(nums)))
            
            a.append(bob)
            a.append(alice)
        return a

File: 3226-MinimumNumberGame/3226-MinimumNumberGame.java
// Last updated: 7/3/2026, 12:49:48 PM
import java.util.Arrays;

class Solution {
    public int[] numberGame(int[] nums) {
        
        Arrays.sort(nums);
        
        for (int i = 0; i<nums.length ;i+=2){
            int temp = nums[i];         // Store nums[i] in a temporary variable
            nums[i] = nums[i + 1];     // Assign nums[i + 1] to nums[i]
            nums[i + 1] = temp; 
        }

        return nums;
    }
}
